### PR TITLE
[EDU-4323] feat: add guide - Next + Turso template

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue:

[EDU-4323]

### Changes

- Add how-to guide to deploy the new Next + Turso template.
- Add links to the guides page.

### Additional links

n/a.

[EDU-4323]: https://aziontech.atlassian.net/browse/EDU-4323?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-turso.mdx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+## Configure o template
+
+Na página do template, clique na aba **Settings** para abrir o formulário e fornecer as informações para configurar sua aplicação.
+
+Os campos identificados com asterisco são obrigatórios.
+
+- **Application Name** *: o nome da sua nova edge application na Azion.
+    - Use um nome único e fácil de lembrar. Se o nome já foi usado, a plataforma retornará uma mensagem de erro.
+- **GitHub Personal Token** *: seu personal token do GitHub.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-turso.mdx:98]:
```suggestion
- **Application Name** *: o nome da sua nova edge application na Azion.
- **GitHub Personal Token** *: seu personal token do GitHub.
- **Turso Database URL** *: a URL do LibSQL  para conetar com seu banco de dados Turso.
- **Turso Authorization Token** *: o token para autorizar o acesso no seu banco de dados Turso.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-turso.mdx [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+## Configure o template
+
+Na página do template, clique na aba **Settings** para abrir o formulário e fornecer as informações para configurar sua aplicação.
+
+Os campos identificados com asterisco são obrigatórios.
+
+- **Application Name** *: o nome da sua nova edge application na Azion.
+- **GitHub Personal Token** *: seu personal token do GitHub.
+- **Turso Database URL** *: a URL do LibSQL  para conetar com seu banco de dados Turso.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-turso.mdx:98]:
```suggestion
- **Turso Database URL** *: a URL do LibSQL para conectar com seu banco de dados Turso.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-turso.mdx [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+### Continuous deployment
+
+Uma vez que o template é implantado, você pode editar e atualizar seus configurações, bem como implementar um workflow de continuous deployment, ou implantação contínua. No entanto, você precisará primeiro *declarar seus secrets no repositório do GitHub do seu projeto* para completar um segundo build com as alterações.
+
+Assim que esta segunda compilação for concluída, você poderá gerenciar seu projeto com um fluxo de trabalho de implantação contínua e editar conforme desejado.
+
+Para fazer isso, abra seu novo repositório do **Next.js App + Configurations + Turso** no GitHub. Em seguida, vá para **Settings** > **Secrets and variables** > **Action** para [adicionar suas variáveis](https://docs.github.com/en/actions/security-guides/encrypted-secrets) seguindo estas instruções:
+
+1. Adicione o Azion personal token *secrets*:
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-turso.mdx:144]:
```suggestion
1. Adicione o Azion personal token aos *secrets*:
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-turso.mdx [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+Agora, seu projeto está pronto para trabalhar com um fluxo de trabalho de implantação contínua, atualizando instantaneamente quaisquer alterações no aplicativo ou no repositório.
+
+### Adicione um domínio personalizado
+
+A edge application criada tem um domínio Azion atribuído para torná-la acessível através do navegador. O domínio tem o seguinte formato: `xxxxxxxxxx.map.azionedge.net`. No entanto, você pode adicionar um domínio personalizado para que os usuários acessem sua aplicação por meio dele.
+
+import Button from '~/components/Button.astro';
+
+<Button href="/pt-br/documentacao/produtos/build/edge-application/domains/" text="saiba mais sobre domínios" variant="secondary" />
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-turso.mdx:174]: em EN leva para um guia

**Review by hannah-sk — APPROVED** [review]

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-turso.mdx [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-turso.mdx:174]:
```suggestion
<Button href="pt-br/documentacao/produtos/guias/configurar-dominio/" text="saiba mais sobre domínios" variant="secondary" /> 
```

File: src/content/docs/en/pages/guides/marketplace/templates/next-turso.mdx [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+
+- **Application Name** *: the name of your edge application on Azion.
+- **GitHub Personal Token** *: your GitHub personal token.
+- **Turso Database URL** *: the LibSQL URL to connect with your Turso database.
+- **Turso Authorization Token** *: the authorization token to access your Turso database.
+
+After completing all the information, click the **Next** button, located in the bottom-right corner. This will start the deployment process.
+
+:::note
+if you provide an invalid GitHub personal token, the creation of the edge application will be executed, but the system won't be able to create the new repository in your GitHub Account. If the modules' subscription is disabled, the deployment will fail.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/guides/marketplace/templates/next-turso.mdx:104]:
```suggestion
If you provide an invalid GitHub personal token, the creation of the edge application will be executed, but the system won't be able to create the new repository in your GitHub Account. If the modules' subscription is disabled, the deployment will fail.
```

**Review by gabriel-azion — APPROVED** [review]

**Pull request merged**